proto/tribune_tts.proto

```diff
@@ -48,13 +60,70 @@ message SynthesizeRequest
 // Provides information to the synthesizer that specifies how to process the request.
 message SynthesizeConfig
 {
-    // Desired sampling frequency in hertz of synthesized audio.
+    // Desired sampling frequency in hertz of synthesized audio. The value 0 means use the default Synthesizer sampling rate.
     int32 sample_rate_hertz = 1;
+
+    // Requested format of the output audio stream.
+    OutputFormat output_format = 2;
```

Comment:
This field should be renamed to `audio_encoding` and also could be moved to `AudioConfig` message

Reply:
Renamed and moved.